README.md

```diff
@@ -8,9 +8,7 @@
 
 [![NPM](https://nodei.co/npm/@node-saml/node-saml.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/@node-saml/node-saml)
 
-This is a [SAML 2.0](http://en.wikipedia.org/wiki/SAML_2.0) authentication provider for Node.js. This was forked from `passport-saml` at v3.0.0 and will become the SAML implementation for `passport-saml`. When this is mature, `passport-saml` will have code removed and replaced by a dependency on this library.
-
-The code was originally based on Michael Bosworth's [express-saml](https://github.com/bozzltron/express-saml) library.
+This is a [SAML 2.0](http://en.wikipedia.org/wiki/SAML_2.0) authentication provider for Node.js.
```

Comment:
@srd90 I cleaned up this paragraph now.
I removed also `This was forked from `passport-saml` at v3.0.0..` because that information is irrelevant.

Also removed `The code was originally based on Michael Bosworth's...` because Michael is in contributors list in package.json file. AND most importantly `express-saml` has been updated previously 11 years ago. So this library reference is irrelevant, too many years has passed.